### PR TITLE
fix: do not require chequebook funding

### PR DESCRIPTION
**Pull request by Cafe137**

### CODE REVIEW

File: src/providers/Bee.tsx [hunk 1]
```diff
@@ -163,14 +163,9 @@ function getStatus(
...
-    if (
-      chequebookAddress?.chequebookAddress &&
-      chequebookBalance !== null &&
-      chequebookBalance?.totalBalance.toBigNumber.isGreaterThan(0)
-    ) {
+    if (chequebookAddress?.chequebookAddress && chequebookBalance !== null) {
       status.chequebook.checkState = CheckState.OK
-    } else if (chequebookAddress?.chequebookAddress) status.chequebook.checkState = CheckState.WARNING
-    else status.chequebook.checkState = CheckState.OK
+    } else status.chequebook.checkState = CheckState.OK
```
Comment (Cafe137) [inline src/providers/Bee.tsx:168]: Not sure why it was like this before

Comment (Cafe137) [reply src/providers/Bee.tsx:168]: Probably because of ultra light node

**Comment by bee-worker** [comment]
## 🐝 PR preview in Swarm

**Preview URL:** https://bah5acgzanrnneajqjf763xhyru2rwbv3umuxaqwrlfputgvek2oz3zwmm5pq.bzz.link
**Swarm Hash:** `6c5ad20130497feddcf88d351b06bba3297042d1595f499aa4569d9de6cc675f`
**Commit Hash:** 49f6f09d1225c648e8f8e3b5fbc3d42563a745b1
**Commit Message:** chore: revert import sorting

**Comment by AuHau** [comment]
How does this work with ultralight node when you do not have actually chequebook deployed? What is shown then?

**Comment by Cafe137** [comment]
> How does this work with ultralight node when you do not have actually chequebook deployed? What is shown then?

The check is not displayed on the status page in that case: 

image

**Review by agazso — APPROVED** [review]
Looks good

**Pull request merged**